File: C/beginner/1933.c
// https://judge.beecrowd.com/pt/problems/view/1933
// Tri-du

// Libraries
#include <stdio.h>

// Main
int main(){
    int a, b, c;

    while(scanf("%d", &a) != EOF){
        
        scanf("%d", &b);

        if (a > b) c = a;
        else if (a < b) c = b;
        else c = a;

        printf("%d\n", c);
    }
    

    return 0;
}

// By: Fernando Serra
// https://github.com/fernandosserra

File: Haskell/beginner/1933.hs
-- https://judge.beecrowd.com/pt/problems/view/1933
-- Tri-du

-- Main
main :: IO ()
main = do
    line <- getLine
    let [a, b] = map read (words line) :: [Int]

    if a > b then
        do let c = a
           print c
    else if b > a then
        do let c = b
           print c
    else
        print a

-- By Fernando Serra
-- https://github.com/fernandosserra